utils/transaction-parser.ts

```diff
@@ -336,6 +336,26 @@ function parseZetaInstruction(ix: PartiallyDecodedInstruction): Instruction {
       };
       break;
 
+    case "placePerpOrder":
+      // price: number;
+      // size: number;
+      // side: Side;
+      // orderType: orderType
+      // clientOrderId: number | undefined;
+      // tag: String | undefined;
+      let placePerpOrderData = decodedIx.data as zetaTypes.placeOrderV3;
```

Comment:
OK to keep placeOrderV3 but might be better to have a separate type for placePerpOrder even though they're the same format, in case we change things later

Reply:
i think i would pref to change it later if / when things change

Reply:
Yeah fine w/ this since this repo is gonna get deprecated anyway